components/myguiplatform/myguirendermanager.cpp

```diff
@@ -126,10 +126,13 @@ class Drawable : public osg::Drawable {
             }
 
             osg::Texture2D* texture = batch.mTexture;
-            if(texture)
+            if(texture && texture->getTextureHeight() > 0 && texture->getTextureWidth() > 0)
```

Comment:
What is this check for? The texture size is set on construction ([here](https://github.com/sandstranger/openmw/blob/dcbb581df134a41c7dd827b94d282fb7b19d7797/components/myguiplatform/myguitexture.cpp#L93) and [here](https://github.com/sandstranger/openmw/blob/dcbb581df134a41c7dd827b94d282fb7b19d7797/components/myguiplatform/myguitexture.cpp#L62)), I can't imagine why it would be 0.